modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-account.adoc
+
+[id="installation-aws-permissions-iam-roles_{context}"]
+= Required AWS permissions for IAM roles
+
+You have the option of defining your own IAM roles that are applied to the instance profiles of your machines created by the installation program. You can specify existing IAM roles by defining the `controlPlane.platform.aws.iamRole` and `compute.platform.aws.iamRoleThis` fields in the `install-config.yaml` file. This allows you to match naming schemes and include predefined permissions boundaries for your IAM roles.
```

Comment:
There's a few instances of "This allows you to..." throughout. IBM SG has an entry on usage of "allow" that might be worth considering.